modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/test/java/com/liferay/dynamic/data/mapping/form/field/type/internal/numeric/NumericDDMFormFieldTemplateContextContributorTest.java

```diff
@@ -94,16 +89,18 @@ public void testGetDataTypeChanged() {
 		DDMFormFieldRenderingContext ddmFormFieldRenderingContext =
 			new DDMFormFieldRenderingContext();
 
+		ddmFormFieldRenderingContext.setLocale(_locale);
```

Comment:
Can't we buy one line by calling `_createDDMFormFieldRenderingContext()`?

Reply:
Sure!